internal/js/modules/k6/browser/common/frame_options.go

```diff
@@ -802,3 +802,30 @@ func (o *FrameWaitForURLOptions) Parse(ctx context.Context, opts sobek.Value) er
 	}
 	return nil
 }
+
+// FrameWaitForResponseOptions are options for Frame.waitForResponse and Page.waitForResponse.
```

Comment:
Maybe best to place in page_options.go if my assumption is correct around `page.waitForResponse`.